doc/openocd.texi

```diff
@@ -11369,15 +11369,15 @@ OpenOCD. When off, they generate a breakpoint exception handled internally.
 The commands below can be used to prevent OpenOCD from using certain RISC-V trigger features.
 For example in cases when there are known issues in the target hardware.
 
-@deffn {Command} {riscv set_enable_eq_match_trigger} [on|off]
+@deffn {Command} {riscv set_enable_eq_match_trigger} [wp|none]
```

Comment:
I would suggest to implement these commands as a single command, taking match type as a parametr.
```suggestion
@deffn {Command} {riscv set_enable_match_trigger_type} [eq|napot|ge_lt] [wp|none]
```

Reply:
Good suggestion, thanks. Addressed.